### PR TITLE
Emby API Key

### PR DESCRIPTION
**Pull request by mousey**
Add the option to read an API Key from the config file and submit that
instead of needing a password. Doesn't remove the use of password so is
backwards compatible with users current config file

### CODE REVIEW

**Comment by sampsyo** [comment]
Thanks for tackling this! :tada:

It looks like there may be some indentation errors in the current commits. You can see the traceback here: https://travis-ci.org/beetbox/beets/jobs/142844029

And the style checker has comments here too: https://travis-ci.org/beetbox/beets/jobs/142844037

Overall, this should be fine if you switch your editor to use 4 spaces for indentation (not real tabs). The Hacking page on the wiki has more details on that.

Could you also please add to the documentation to describe how to use this option? (And add a quick note to the changelog?)

**Comment by sampsyo** [comment]
Awesome! This looks good, and the tests are passing. For the documentation, I also meant to suggest adding to the plugin documentation itself—i.e., `embyupdate.rst`—in addition to the changelog.

File: docs/plugins/embyupdate.rst [hunk 1]
```diff
@@ -30,4 +31,5 @@ The available options under the ``emby:`` section are:
 - **port**: The Emby server port.
   Default: 8096
 - **username**: A username of a Emby user that is allowed to refresh the library.
-- **password**: That user's password.
+- **apikey**: An Emby API key
+- **password**: If no API key is present then enter the user aboves password.
```
Comment (jrobeson) [inline docs/plugins/embyupdate.rst]: this needs rewording. Perhaps something like.  Emby password (only needed if no API key is provided)

**Comment by sampsyo** [comment]
Thanks, @mousey! And I agree, @jrobeson—I've reworded the documentation stuff a bit (and merged).

**Pull request merged**